### ISSUE TITLE
[Bug] Accessibility: Tab collection items cannot be expanded or closed with TalkBack

### ISSUE THREAD

**Issue by MarcoZehe** [mozilla-mobile/fenix#8996]
## Steps to reproduce

1. Add tabs to a collection.
2. Open a New Tab panel.
3. Start TalkBack.
4. Try to interact with the collection by first focusing, then double-tapping it.

### Expected behavior

TalkBack announces that the collection is expandable.
The collection expands, ready to be interacted with.

### Actual behavior

TalkBack does not announce that a collection is expandable.
The collection doesn't expand on double-tapping.

### Device information

* Android device: ? Pixel 3.
* Fenix version: ? Nightly.

CC @eeejay

**Comment by sv-sdeiac** [comment]
Hi @MarcoZehe, I can expand the collections using the double-tapping on the talkback but indeed, no announcements when I expand the collection. 

Device:
- Google Pixel (Android Q).

Build:
- Firefox Preview Nightly 3/5 #20650606.

**Comment by asadotzler** [comment]
What's happening here is that focus is lost when you double tap to expand the collection. I can swipe back to the collection after the focus is lost and have the collection contents be read to me and can be closed (but once again focus is lost when the collection closes.)

This is an accessibility P2. The feature is available and works but is difficult to use because of focus behavior. This kind of bug prevents Fenix from being a delightful experience for users with disabilities.

**Comment by eliserichards** [comment]
This might be on hold since we are moving collections into the tabs tray.
